### ISSUE TITLE
efivars mounted but "Required efivarfs filesystem was not found"

### ISSUE THREAD

**Issue by eoli3n** [fwupd/fwupd#3255]
**Describe the bug**
When i run ``fwupdmgr get-devices`` I get ``Required efivarfs filesystem was not found`` but mounting it manually doesn't solve the warning.

**Steps to Reproduce**
```
$ mount -t efivarfs efivarfs /sys/firmware/efi/efivars
$ fwupdmgr get-devices
# cry
```

**Expected behavior**
fwupdmgr to find efivars

**fwupd version information**
```shell
$ fwupdmgr --version
client version:	1.5.8
compile-time dependency versions
	gusb:	0.3.5

daemon version:	1.5.8
```

Please note how you installed it (`apt`, `dnf`, `pacman`, source, etc): xbps

**fwupd device information**
```shell
$ fwupdmgr get-devices --show-all-devices
WARNING: Required efivarfs filesystem was not found
  See https://github.com/fwupd/fwupd/wiki/PluginFlag:efivar-not-mounted for more information.
Latitude 7490
│
├─Dell WD15:
│ │   Device ID:          3086ec033384094e82dff9652509f61e12f66da7
│ │   Résumé:             A USB type-C docking station
│ │   Version actuelle:   0.0.0
│ │   Fournisseur:        Dell Inc. (PCI:0x1028)
│ │   Update Error:       Not updatable as efivarfs was not found
│ │   GUID:               e7ca1f36-bf73-4574-afe6-a4ccacabf479
│ │   Device Flags:       • System requires external power source
│ │                       • Needs a reboot after installation
│ │ 
│ ├─Dell WD15 Passive Cable:
│ │     Device ID:        09821649fd4c49c71adcaadc9f779db402b46cb2
│ │     Résumé:           A USB type-C docking station
│ │     Version actuelle: 0.3.18
│ │     Fournisseur:      Dell Inc. (PCI:0x1028)
│ │     Update Error:     Not updatable as efivarfs was not found
│ │     GUID:             fece1537-d683-4ea8-b968-154530bb6f73
│ │     Device Flags:     • System requires external power source
│ │                       • Needs a reboot after installation
│ │   
│ ├─Dell WD15 Port Controller 1:
│ │     Device ID:        5da6f3f1074a5fd9e04d426b49b301310515e34f
│ │     Résumé:           A USB type-C docking station
│ │     Version actuelle: 1.1.8
│ │     Fournisseur:      Dell Inc. (PCI:0x1028)
│ │     Update Error:     Not updatable as efivarfs was not found
│ │     GUID:             8ba2b709-6f97-47fc-b7e7-6a87b578fe25
│ │     Device Flags:     • System requires external power source
│ │                       • Needs a reboot after installation
│ │   
│ └─VMM3332 inside Dell WD15/TB16/TB18 wired Dock:
│       Device ID:        d762543f8c20f636e6fff031a000078d3e10c600
│       Résumé:           Multi-Stream Transport Device
│       Version actuelle: 3.12.02
│       Fournisseur:      Synaptics (DRM_DP_AUX_DEV:0x06CB)
│       GUIDs:            87f8cd71-1e6b-5622-94e9-0e9d11bdde33 ← MST-274
│                         c1e81f96-234e-597d-b9cb-6477d0652868 ← MST-wd15-274
│                         78c73924-821a-5e6f-af82-86df6f67b182 ← MST-wd15-vmm3332-274
│                         c1e2bea5-4cf2-5f84-b965-533e8e5fea45 ← MST-tb16-274
│                         02b77ccb-03ab-521f-b98e-ef05395e9fa2 ← MST-tb16-vmm3332-274
│                         82460515-30d3-5cdf-be4c-571340842a10 ← MST-tb18-274
│                         a6f00ca7-54b7-5d51-9d6e-543d8df85376 ← MST-tb18-vmm3332-274
│                         d6072785-6fc0-5f83-9d49-11376e7f48b1 ← MST-leaf
│                         55d7e858-0ce0-506f-afb3-e8f6d77bf506 ← MST-leaf-vmm3332-274
│                         4e4e06b6-8217-5cf0-8eff-ce5e987d531a ← MST-leaf-274
│       Device Flags:     • Updatable
│     
├─Intel(R) Core™ i5-8250U CPU @ 1.60GHz:
│     Device ID:          4bde70ba4e39b28f9eab1628f9dd6e6244c03027
│     Fournisseur:        Intel
│     GUIDs:              b9a2dd81-159e-5537-a7db-e7101d164d3f ← cpu
│                         30249f37-d140-5d3e-9319-186b1bd5cac3 ← CPUID\PRO_0&FAM_06
│                         561403e8-143a-5071-ab09-bf5e4c146983 ← CPUID\PRO_0&FAM_06&MOD_8E
│                         54bdc429-0179-5019-9cb6-8a343324c97c ← CPUID\PRO_0&FAM_06&MOD_8E&STP_A
│     Device Flags:       • Périphérique interne
│   
├─System Firmware:
│ │   Device ID:          38390d47b6b0e59dab94e5feafcd6adefefdb76e
│ │   Version actuelle:   70144
│ │   Version minimum:    70144
│ │   Fournisseur:        Dell Inc. (DMI:Dell Inc.)
│ │   Update Error:       Not updatable as efivarfs was not found
│ │   GUIDs:              a86a3f07-b7d3-4545-9238-0274fc1ba682
│ │                       230c8b18-8d9b-53ec-838b-6cfc0383493a ← main-system-firmware
│ │                       ded317a0-d760-5d47-b807-0ac7af13d84b ← UEFI\RES_{A86A3F07-B7D3-4545-9238-0274FC1BA682}
│ │   Device Flags:       • Périphérique interne
│ │                       • System requires external power source
│ │                       • Needs a reboot after installation
│ │                       • Cryptographic hash verification is available
│ │                       • Device is usable for the duration of the update
│ │ 
│ └─Intel AMT [unprovisioned]:
│       Device ID:        e2623122c99d58220498aacbfcfdb1baebbae3c5
│       Résumé:           Hardware and firmware technology for remote out-of-band management
│       Version actuelle: 11.8.82.3838
│       Version du chargeur d’amorçage:11.8.82.3838
│       Fournisseur:      Intel Corporation
│       GUID:             2800f812-b7b4-2d4b-aca8-46e0ff65814c
│       Device Flags:     • Périphérique interne
│     
├─TOSHIBA KSG60ZMV256G M.2 2280 256GB:
│     Device ID:          64ec4cd1e1c9565e79b00f4e7221b1c689b33e96
│     Résumé:             ATA Drive
│     Version actuelle:   ABDA4102
│     Fournisseur:        Toshiba (ATA:0x1179|OUI:00080d)
│     Serial Number:      681B86PJK5SP
│     GUIDs:              5c9bd035-5fdf-521d-ad4c-1f6562fbded7 ← IDE\TOSHIBA_KSG60ZMV256G_M.2_2280_256GB_____ABDA4102
│                         d2751458-58a0-58cd-b6f6-ebe88d68df08 ← IDE\0TOSHIBA_KSG60ZMV256G_M.2_2280_256GB_____
│                         20bb3a66-e85a-5fdd-862d-be4d2bd6f383 ← TOSHIBA KSG60ZMV256G M.2 2280 256GB
│     Device Flags:       • Périphérique interne
│                         • Updatable
│                         • System requires external power source
│                         • Needs a reboot after installation
│                         • Device is usable for the duration of the update
│   
├─TPM:
│ │   Device ID:          c6a80ac3a22083423992a3cb15018989f37834d6
│ │   Version actuelle:   7.2.0.1
│ │   Fournisseur:        Nuvoton Technology (TPM:NTC|PCI:0x1028)
│ │   GUIDs:              ff71992e-52f7-5eea-94ef-883e56e034c6 ← system-tpm
│ │                       fac1c8f3-73c8-5cd6-8330-07a3690b5140 ← TPM\VEN_NTC&DEV_0000
│ │                       4521fefa-4a2f-537f-a7f9-d353841eb18e ← TPM\VEN_NTC&MOD_NPCT75x6rls
│ │                       e9ccc1dc-960a-5e09-afe9-e59a904b776d ← TPM\VEN_NTC&DEV_0000&VER_2.0
│ │                       c05f797a-8d97-547f-a7a3-c5c0339748e9 ← TPM\VEN_NTC&MOD_NPCT75x6rls&VER_2.0
│ │   Device Flags:       • Périphérique interne
│ │ 
│ └─Event Log:
│       Device ID:        58bd405f31c48e6eca290b425f530a94c91e955c
│       GUID:             a25657fe-b5dc-5be0-8b78-8b9dfec678ff ← system-tpm-eventlog
│       Device Flags:     • Périphérique interne
│     
└─UHD Graphics 620:
      Device ID:          5792b48846ce271fab11c4a545f7a3df0d36e00a
      Version actuelle:   07
      Fournisseur:        Intel Corporation (PCI:0x8086)
      GUIDs:              8b1ece07-ae57-551b-9184-720012c0c6ad ← PCI\VEN_8086&DEV_5917&SUBSYS_1028081C&REV_07
                          121c97d6-9509-5e08-90f1-78a75d1ac748 ← PCI\VEN_8086&DEV_5917&SUBSYS_1028081C
                          fed2efa4-7045-55a1-b4fc-b29283d59fe5 ← PCI\VEN_8086&DEV_5917&REV_07
                          8b72a10c-1279-5f8e-a28a-34fb11a58240 ← PCI\VEN_8086&DEV_5917
      Device Flags:       • Périphérique interne
                          • Cryptographic hash verification is available
```

**error logs**
```
root@osz ~# fwupdmgr get-devices > /dev/null
WARNING: Required efivarfs filesystem was not found
  See https://github.com/fwupd/fwupd/wiki/PluginFlag:efivar-not-mounted for more information.

root@osz ~# ls /sys/firmware/efi/efivars/

root@osz ~# mount -t efivarfs efivarfs /sys/firmware/efi/efivars

root@osz ~# ls /sys/firmware/efi/efivars/
AbtStatus-a0b1889e-00eb-445b-8ca9-e91ce43c907d
AmiEntryS3Addr-074e1e48-8132-47a1-8c2c-3f14ad9a66dc
AmiGopOutputDp-01368881-c4ad-4b1d-b631-d57a8ec8db6b
AMITCGPPIVAR-a8a2093b-fefa-43c1-8e62-ce526847265e
AMITSESetup-c811fa38-42c8-4579-a9bb-60e94eddfb34
AmtWrapperKvmSolFlag-972e2031-6ebf-4535-abd6-3654ea409510
AssetTag-01368881-c4ad-4b1d-b631-d57a8ec8db6b
AuditMode-8be4df61-93ca-11d2-aa0d-00e098032b8c
BiosGuardCapsuleVariable-368b3153-563d-4610-8d94-47a9fa8c4c16
BiosGuardRecoveryAddressVariable-368b3152-563d-4670-8d94-47a9fa8c4c16
Boot0000-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0001-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0002-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0003-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0004-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0005-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0006-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0007-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot0008-8be4df61-93ca-11d2-aa0d-00e098032b8c
Boot000A-8be4df61-93ca-11d2-aa0d-00e098032b8c
BootCurrent-8be4df61-93ca-11d2-aa0d-00e098032b8c
BootFFF6-5990c250-676b-4ff7-8a0d-529319d0b254
BootFFF7-5990c250-676b-4ff7-8a0d-529319d0b254
BootFFF8-5990c250-676b-4ff7-8a0d-529319d0b254
BootFFFB-5990c250-676b-4ff7-8a0d-529319d0b254
BootFFFC-5990c250-676b-4ff7-8a0d-529319d0b254
BootFFFD-5990c250-676b-4ff7-8a0d-529319d0b254
BootFFFE-5990c250-676b-4ff7-8a0d-529319d0b254
BootList-65cbd9d9-ab77-4a61-b288-2763405d588a
BootOptionSupport-8be4df61-93ca-11d2-aa0d-00e098032b8c
BootOrder-8be4df61-93ca-11d2-aa0d-00e098032b8c
COMPAL-b697de83-1ab6-42c4-9dee-a806c637818b
ConIn-8be4df61-93ca-11d2-aa0d-00e098032b8c
ConInDev-8be4df61-93ca-11d2-aa0d-00e098032b8c
ConOut-8be4df61-93ca-11d2-aa0d-00e098032b8c
ConOutDev-8be4df61-93ca-11d2-aa0d-00e098032b8c
CpuSetupVolatileData-b08f97ff-e6e8-4193-a997-5e9e9b0adb32
CpuSmm-90d93e09-4e91-4b3d-8c77-c82ff10e3c81
CurrentPolicy-77fa9abd-0359-4d32-bd60-28f4e78f784b
db-d719b2cb-3d3a-4596-a3bc-dad00e67656f
dbDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c
dbx-d719b2cb-3d3a-4596-a3bc-dad00e67656f
dbxDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c
DefaultBootOrder-45cf35f6-0d6e-4d04-856a-0370a5b16f53
DellBrightnessValue-ec87d643-eba4-4bb5-a1e5-3f3e36b20da9
DellMonotonicCounter-01368881-c4ad-4b1d-b631-d57a8ec8db6b
DellSecureBootMode-7b59104a-c00d-4158-87ff-f04d6396a915
DellVar01-e224eaa0-4358-6ac8-3cce-daa44e54f638
DeployedMode-8be4df61-93ca-11d2-aa0d-00e098032b8c
DeploymentModeNv-97e8965f-c761-4f48-b6e4-9ffa9cb2a2d6
DIAGEEPROM_VAR-8ebe3d07-3420-4bfa-8c13-3a4e0fae6860
dump-type0-10-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-1-1-1615190360-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-2-1-1615190360-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-3-1-1615190360-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-4-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-5-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-6-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-7-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-8-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
dump-type0-9-1-1615190361-D-cfc8fc79-be2e-4ddc-97f0-9f98bfe298a0
Ep-73dad563-8f27-42af-918f-8651eb0a93ef
EPCBIOS-c60aa7f6-e8d6-4956-8ba1-fe26298f5e87
ePSAVersion-66b36b33-8094-424d-ba45-e876d62c45c1
ErrOut-8be4df61-93ca-11d2-aa0d-00e098032b8c
ErrOutDev-8be4df61-93ca-11d2-aa0d-00e098032b8c
EvaluateDefaults4FirstBoot2-4b4d31be-88c6-446a-a4a9-4ad0f612b32e
FactoryDefaults-c54906f9-eb09-4457-a007-4154652fcfa5
FastBootOption-b540a530-6978-4da7-91cb-7207d764d262
FPDT_Volatile-01368881-c4ad-4b1d-b631-d57a8ec8db6b
fwupd-a86a3f07-b7d3-4545-9238-0274fc1ba682-0-0abba7dc-e516-4167-bbf5-4d9d1c739416
GNVS_PTR-0a602c5b-05a0-40c4-9181-edcd891d0003
GsetLegacyIplDefaultValue-3a21751e-bd32-4825-8754-82a47f01b09b
GsetUefiIplDefaultValue-7f3301c7-2405-4765-aa2e-d9ed28aea950
IntUcode-eda41d22-7729-5b91-b3ee-ba619921cefa
KEK-8be4df61-93ca-11d2-aa0d-00e098032b8c
KEKDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c
Kernel_ATPSiStatus-77fa9abd-0359-4d32-bd60-28f4e78f784b
Kernel_EntRevokeSiStatus-77fa9abd-0359-4d32-bd60-28f4e78f784b
Kernel_RvkSiStatus-77fa9abd-0359-4d32-bd60-28f4e78f784b
Kernel_SiStatus-77fa9abd-0359-4d32-bd60-28f4e78f784b
Kernel_SkuSiStatus-77fa9abd-0359-4d32-bd60-28f4e78f784b
Kernel_WinSiStatus-77fa9abd-0359-4d32-bd60-28f4e78f784b
KvmActive-f7f70708-b3c0-4be0-b00c-ae91da4adc03
LastKnownGoodConfig-c54906f9-eb09-4457-a007-4154652fcfa5
LastModeState-2d2edd10-1661-47e3-bdff-581f2a63ec0d
LoaderEntryDefault-4a67b082-0a4c-41cf-b6c7-440b29bb8c4f
LoaderSystemToken-4a67b082-0a4c-41cf-b6c7-440b29bb8c4f
Logo_resolutionX-972e2031-6ebf-4535-abd6-3654ea409510
Logo_resolutionY-972e2031-6ebf-4535-abd6-3654ea409510
MaximumTableSize-4b3082a3-80c6-4d7e-9cd0-583917265df1
MemoryOverwriteRequestControl-e20939be-32d4-41be-a150-897f85d49829
MemoryOverwriteRequestControlLock-bb983ccf-151d-40e1-a07b-4a17be168292
MeSetupStorage-5432122d-d034-49d2-a6de-65a829eb4c74
MonotonicCounter-01368881-c4ad-4b1d-b631-d57a8ec8db6b
NBGopPlatformData-ec87d643-eba4-4bb5-a1e5-3f3e36b20da9
NetworkStackVar-d1405d16-7afc-4695-bb12-41459d3695a2
OfflineUniqueIDEKPubCRC-eaec226f-c9a3-477a-a826-ddc716cdc0e3
OfflineUniqueIDEKPub-eaec226f-c9a3-477a-a826-ddc716cdc0e3
OsIndications-8be4df61-93ca-11d2-aa0d-00e098032b8c
OsIndicationsSupported-8be4df61-93ca-11d2-aa0d-00e098032b8c
OsType-a66919d2-6c45-403e-b00a-9bce58e97315
PBRDevicePath-a9b5f8d2-cb6d-42c2-bc01-b5ffaae4335e
PCI_COMMON-aca9f304-21e2-4852-9875-7ff4881d67a5
PK-8be4df61-93ca-11d2-aa0d-00e098032b8c
PKDefault-8be4df61-93ca-11d2-aa0d-00e098032b8c
PlatformLang-8be4df61-93ca-11d2-aa0d-00e098032b8c
PlatformLangCodes-8be4df61-93ca-11d2-aa0d-00e098032b8c
PreMonotoicCount-65cbd9d9-ab77-4a61-b288-2763405d588a
RstOptaneConfig-4da4f952-2516-4d06-8975-65036403a8c7
RstSataFastEnumVar-193dfefa-a445-4302-99d8-ef3aad1a04c6
SataPortNumber-01368881-c4ad-4b1d-b631-d57a8ec8db6b
SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c
SecureBootSetup-7b59104a-c00d-4158-87ff-f04d6396a915
ServiceTag-01368881-c4ad-4b1d-b631-d57a8ec8db6b
SetUpdateCountVar-81c76078-bfde-4368-9790-570914c01a65
Setup-ec87d643-eba4-4bb5-a1e5-3f3e36b20da9
SetupMode-8be4df61-93ca-11d2-aa0d-00e098032b8c
SIDSUPPORT-7d3dceee-cbce-4ea7-8709-6e552f1edbde
SignatureSupport-8be4df61-93ca-11d2-aa0d-00e098032b8c
SMBIOS_ENTRY_ADDR-0a602c5b-05a0-40c4-9181-edcd891d0036
SmbiosEntryPointTable-4b3082a3-80c6-4d7e-9cd0-583917265df1
SmbiosScratchBuffer-4b3082a3-80c6-4d7e-9cd0-583917265df1
SmbiosV3EntryPointTable-4b3082a3-80c6-4d7e-9cd0-583917265df1
SOFTWAREGUARDSTATUS-9cb2e73f-7325-40f4-a484-659bb344c3cd
SolActive-f7f70708-b3c0-4be0-b00c-ae91da4adc03
StdDefaults-4599d26f-1a11-49b8-b91f-858745cff824
TbtHRStatusVar-ba1d893b-803e-4b26-a3de-585703ff7bd6
TbtPresentStatusOnPost-ec87d643-eba4-4bb5-a1e5-3f3e36b20da9
Timeout-8be4df61-93ca-11d2-aa0d-00e098032b8c
TPMPERBIOSFLAGS-7d3dceee-cbce-4ea7-8709-6e552f1edbde
TpmServFlags-7d3dceee-cbce-4ea7-8709-6e552f1edbde
UnlockIDCopy-eaec226f-c9a3-477a-a826-ddc716cdc0e3
UsbSupport-ec87d643-eba4-4bb5-a1e5-3f3e36b20da9
UserBootOrder-65cbd9d9-ab77-4a61-b288-2763405d588a
UserBootOrderCount-65cbd9d9-ab77-4a61-b288-2763405d588a
UserDefaults-c54906f9-eb09-4457-a007-4154652fcfa5
VendorKeys-8be4df61-93ca-11d2-aa0d-00e098032b8c

root@osz ~# fwupdmgr get-devices > /dev/null
WARNING: Required efivarfs filesystem was not found
  See https://github.com/fwupd/fwupd/wiki/PluginFlag:efivar-not-mounted for more information.
```

**System UEFI configuration**
Please provide the output of the following commands:
```shell
$ efibootmgr -v
BootCurrent: 000A
Timeout: 2 seconds
BootOrder: 000A,0002,0003
Boot0001* Linux Firmware Updater	HD(1,GPT,b8b60cdf-31ec-46bb-80c1-9dadac461b8a,0x800,0x100000)/File(\EFI\arch\fwupdx64.efi)
Boot0002* ZFSBootMenu Backup	HD(1,GPT,4da1486d-da45-43ac-b1c8-7973ff620c70,0x800,0x100000)/File(\EFI\ZBM\vmlinuz-backup.efi)r.o.o.t.=.z.f.s.b.o.o.t.m.e.n.u.:.P.O.O.L.=.z.r.o.o.t. .r.o. .q.u.i.e.t. .s.p.l._.h.o.s.t.i.d.=.7.4.e.1.6.e.c.6.
Boot0003* Archlinux Netboot	PciRoot(0x0)/Pci(0x17,0x0)/Sata(2,65535,0)/HD(1,GPT,b8b60cdf-31ec-46bb-80c1-9dadac461b8a,0x800,0x100000)/File(\EFI\netboot\ipxe.efi)
Boot000A* ZFSBootMenu	HD(1,GPT,4da1486d-da45-43ac-b1c8-7973ff620c70,0x800,0x100000)/File(\EFI\ZBM\vmlinuz.efi)r.o.o.t.=.z.f.s.b.o.o.t.m.e.n.u.:.P.O.O.L.=.z.r.o.o.t. .r.o. .q.u.i.e.t. .s.p.l._.h.o.s.t.i.d.=.7.4.e.1.6.e.c.6.
```

```shell
$ efivar -l | grep fw
0abba7dc-e516-4167-bbf5-4d9d1c739416-fwupd-a86a3f07-b7d3-4545-9238-0274fc1ba682-0
```

```shell
$ tree /boot
/boot
├── config-5.11.18_1
├── config-5.11.21_1
├── EFI
│   └── gummiboot
├── initramfs-5.11.18_1.img
├── initramfs-5.11.21_1.img
├── vmlinuz-5.11.18_1
└── vmlinuz-5.11.21_1
```

**Additional questions**
- Operating system and version: latest voidlinux
- Have you tried rebooting? yep
- Is this a regression? nop
- Are you using an NVMe disk? yep TOSHIBA_KSG60ZMV256G_M.2_2280_256GB
- Is secure boot enabled? nop
- Is this a Lenovo system with 'Boot Order Lock' turned on in the BIOS? nop

**Comment by hughsie** [comment]
> $ mount -t efivarfs efivarfs /sys/firmware/efi/efivars
> $ fwupdmgr get-devices

Maybe try restarting the fwupd service before the get-devices?

**Comment by eoli3n** [comment]
it seems that it made ``fwupd`` not build properly, because of a test failing
https://github.com/void-linux/void-packages/blob/master/srcpkgs/fwupd/template#L54

``efivar`` was not installed before I wrote that issue, I noticed the miss when running the command you ask in issue template.

The template doesn't contains any service, but i can find "/usr/libexec/fwupd/fwupd" running.
Killing it manually and starting it again solved the problem.

Thanks
